install_config/cluster_metrics.adoc

```diff
@@ -288,6 +288,48 @@ additional flag to the metrics-deployer: `DYNAMICALLY_PROVISION_STORAGE=true`.
 You can use EBS, GCE, and Cinder storage back-ends to
 xref:../install_config/persistent_storage/dynamically_provisioning_pvs.adoc#install-config-persistent-storage-dynamically-provisioning-pvs[dynamically provision persistent volumes].
 
+In tests performed with with 210 and 990 {product-title} nodes, where 10500 pods
+and 11000 pods were monitored respectively, the cassandra database grew at the
+speed as showed in the table below:
+
+.Cassandra Database storage requirements based on number of nodes/pods in the cluster
+[options="header"]
+|===
+|Number of Nodes |Number of Pods |Cassandra Storage growth speed |Cassandra storage growth per day |Cassandra storage growth per week
+
+|210
+|10500
+|500MB per hour
+|15 GB
+|75 GB
+
+|990
+|11000
+|1 GB per hour
+|30 GB
+|210 GB
+|===
+
+In the above calculation, approximately 20% of the expected size was added as
+overhead to ensure that the storage requirements do not exceed calculated value.
+
+If the `METRICS_DURATION` and `METRICS_DURATION` values are kept at the default
+(`7` days and `15` seconds respectively), it is safe to plan Cassandra storage
+size requrements for week, as in the values above.
+
+[WARNING]
+====
+Because {product-title} metrics uses Cassandra database as datastore for metrics data, if `USE_PERSISTANT_STORAGE=true` is set during the metrics setup process, `PV` will be on top in the network storage, with NFS as the default. However, using network storage in combination with Cassandra is
+not recommended, as per the http://docs.datastax.com/en/archived/cassandra/1.2/cassandra/architecture/architecturePlanningAntiPatterns_c.html[Cassandra documentation].
+====
+
+*Known issues and limitations*
+When scaling above 12000 pods, data in metrics graphs is not visible. This is related to the `heapster` metrics component not being able to handle load
+related to mentioned pods numbers in {product-title} cluster.
+
+This issue is tracked in https://bugzilla.redhat.com/show_bug.cgi?id=1388815[Red Hat Bugzilla] and it is
```

Comment:
Not sure if we link to BZs in our docs?  If so, it's fine.  Just wondering.  Also, with the re-phrased sentence above, perhaps you can collapse this sentence somehow.